src/bot/commands/bonus/role.ts

```diff
@@ -189,14 +188,14 @@ async function betaSystem(
   interaction.client.logger.debug(`New role give affected ${affected} members`);
 
   await interaction.followUp({
-    content: oneLine`Successfully gave \`${changeAmount}\` bonus XP
-      to \`${affected}\` member${affected == 1 ? '' : 's'} with role ${role}`,
+    content: `${t(bonus.success)} \`${changeAmount}\` ${t(bonus.bonusXP)} \`${affected}\` ${t(bonus.withRole)} ${role}`,
```

Comment:
a few of these `t()` functions are missing quotes around the keys: `t(bonus.success)` is invalid, it should be `t('bonus.success')`